### PR TITLE
Add badges in README

### LINKED ISSUE #862: Makeup README
It is good to add some badges to README
- CI status
- Code coverage
- ...

### PR DESCRIPTION
**Pull request by sh-cho**
Added some badges in README

- CI workflow status
- Code coverage
- Latest release version (GitHub release)
- ~Slack workspace~ -> Discord Server

Closes #862

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@ Visit [the official web site](https://line.github.io/centraldogma/) for more inf
 
 # Central Dogma
 
+[![CI](https://github.com/line/centraldogma/actions/workflows/actions_build.yml/badge.svg?branch=main&event=push)](https://github.com/line/centraldogma/actions/workflows/actions_build.yml)
+[![codecov.io](https://codecov.io/github/line/centraldogma/coverage.svg?branch=main)](https://codecov.io/github/line/centraldogma?branch=main)
+[![Latest Release Version](https://img.shields.io/github/v/release/line/centraldogma)](https://github.com/line/centraldogma/releases/latest)
+[![Slack workspace](https://img.shields.io/badge/join-slack-orange?logo=slack)](https://join.slack.com/t/central-dogma/shared_invite/enQtNjA5NDk5MTExODQzLWFhOWU2NGZhNDk3MjBmNzczZDYyZjRmMTI1MzdiNGI3OTcwNWZlOTkyY2U3Nzk4YTM2NzQ2NGJhMjQ1NzJlNzQ)
```
Comment (minwoox) [inline README.md:8]: We have moved to Discord: https://armeria.dev/s/discord
Could you fix that, please? 😄

**Review by minwoox — COMMENTED** [review]
Oops, I missed this PR.
Thanks a lot, @sh-cho! 😉

File: README.md (hunk 1 above)
Comment (sh-cho) [reply README.md]: Fixed :)

**Review by minwoox — APPROVED** [review]
Thanks a lot, @sh-cho! 🚀 👍 💯 
Are you using Central Dogma by the way? 😆

**Comment by sh-cho** [comment]
> Thanks a lot, @sh-cho! 🚀 👍 💯 Are you using Central Dogma by the way? 😆

Sadly not now 🥲

**Comment by minwoox** [comment]
@sh-cho Hope you can use this someday. 😆

**Review by jrhee17 — APPROVED** [review]
👍 👍 👍

**Review by ikhoon — APPROVED** [review]
Thanks, @sh-cho! 🙇‍♂️

**Review by trustin — APPROVED** [review]
Thank you very much for keeping it up-to-date, @sh-cho! :bow:

**Pull request merged**